transformations/GermanGenderSwap/transformation.py

```diff
@@ -0,0 +1,2433 @@
+    "Uberlebender":"Uberlebende",
+    "Einwohner":"Einwohnerin",
+    "Bewohner":"Bewohnerin",
+    "Forscher":"Forscherin",
+    "Programmierer":"Programmiererin",
+    "Professor":"Professorin",
+    "Dozent":"Dozentin",
+    "Begleiter":"Begleiterin",
+    "Besucher":"Besucherin",
+    "Theologe":"Theologin",
+    "Immigrant":"Immigrantin",
+    "Farmer":"Farmerin",
+    "Senator":"Senatorin",
+    "Wanderer":"Wanderin",
+    "Kurator":"Kuratorin",
+    "Wachter":"Wachterin",
+    "Treuhander":"Treuhanderin",
+    "Huter":"Huterin",
+    "Spion":"Spionin",
+    "Klager":"Klagerin",
+    "Antragsteller":"Antragstellerin",
+    "Dieb":"Diebin",
+    "Pazifist":"Pazifistin",
+    "Metzger":"Metzgerin",
+    "Verfuhrer":"Verfihrerin",
+    "Kanzler":"Kanzlerin",
+    "Gewinner":"Gewinnerin",
+    "Einsiedler":"Einsiedlerin",
+    "Prostituierter":"Prostituierte",
+    "Gastronom":"Gastronomin",
+    "Laufer":"Lauferin",
+    "Bote":"Botin",
+    "Journalist":"Journalistin",
+    "Publizist":"Publizistin",
+    "Vegetarier":"Vegetarierin",
+    "Schwimmer":"Schwimmerin",
+    "Uhrmacher":"Uhrmacherin",
+    "Lugner":"Lugnerin",
+    "Schwindler":"Schwindlerin",
+    "Stripper":"Stripperin",
+    "Maler":"Malerin",
+    "Richter":"Richterin",
+    "Experte":"Expertin",
+    "Komponist":"Komponistin",
+    "Trainer":"Trainerin",
+    "Tutor":"Tutorin",
+    "Erzieher":"Erzieherin",
+    "Chiropraktiker":"Chiropraktikerin",
+    "Blogger":"Bloggerin",
+    "Aktivist":"Aktivistin",
+    "Banker":"Bankerin",
+    "Biograph":"Biographin",
+    "Pilot":"Pilotin",
+    "Schiedsrichter":"Schiedsrichterin",
+    "Abenteurer":"Abenteurerin",
+    "Schneider":"Schneiderin",
+    "Ubersetzer":"Ubersetzerin",
+    "Verrater":"Verraterin",
+    "Sunder":"Sunderin",
+    "Bildhauer":"Bildhauerin",
+    "Politiker":"Politikerin",
+    "Philosoph":"Philosophin",
+    "Ornithologe":"Ornithologin",
+    "Motorradfahrer":"Motorradfahrerin",
+    "Radfahrer":"Radfahrerin",
+    "Moderator":"Moderatorin",
+    "Missionar":"Missionarin",
+    "Kuppler":"Kupplerin",
+    "Mikrobiologe":"Mikrobiologin",
+    "Holzfaller":"Holzfallerin",
+    "Interviewer":"Interviewerin",
+    "Zigeuner":"Zigeunerin",
+    "Gitarrist":"Gitarristin",
+    "Wahrsager":"Wahrsagerin",
+    "Fischhandler":"Fischhandlerin",
+    "Taucher":"Taucherin",
+    "Springer":"Springerin",
+    "Diplomat":"Diplomatin",
+    "Stellvertreter":"Stellvertreterin",
+    "Masseur":"Masseuse",
+    "Neffe":"Nichte",
+    "Lowe":"Lowin", #animals 
+    "Kater":"Katze"
+}
+
+def remove_punc(my_str):
+  punctuations = '''!()-[]{};:\,<>./?@#$%^&*_~'''
+  no_punct = ""
+  for char in my_str:
+   if char not in punctuations:
+       no_punct = no_punct + char
+  return no_punct 
+
+
+def preserve_capitalization(original, transformed):
+  if original[0].isupper():
+    transformed = transformed.capitalize()
+    return transformed
+  else:
+    return transformed
+ 
+#CHECK IF PRECEEDING WORD NEEDS TO BE REPLACED
```

Comment:
no need to shout. Also add space and terminate comment with a period.

Reply:
I've removed these capitalized comments.